NppJSONViewer/NppJsonViewer/JsonViewDlg.cpp

```diff
@@ -76,6 +77,30 @@ void JsonViewDlg::FormatJson()
     }
     else
     {
+        if (m_pSetting->parseOptions.bReplaceUndefined)
+        {
+            auto text = selectedText.substr(res.error_pos, 9);
+            std::transform(text.begin(), text.end(), text.begin(), [](unsigned char c) { return (unsigned char)std::tolower(c); });
+            if (text == "undefined")
+            {
+                try
+                {
+                    std::regex regex("(\\:)([\\s]*?)undefined([\\s,}]*?)", std::regex_constants::icase);
```

Comment:
regex does not work if `undefined` is part of an array. Consider below example.

```
{
"Nan": NaN,
"Inf":[-Infinity, Infinity, -Inf, Inf],
"UndefinedArr":[undefined, undefined, undefined],
"Undefined1": undefined,
"Undefined2": undefined,
"UndefinedStr": "undefined"
}
```

Reply:
Now this case works too.